### PR TITLE
Add Quartz and JBehave

### PR DESCRIPTION
**Pull request by tracer021**
Add quartz and jbehave. Quartz is a well known job scheduler library for java, while JBehave is a BDD framework created by Dan North.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JavaServer Pages](https://jsp.java.net/) - Common templating for websites with custom tag libraries.
 * [JUnit](http://junit.org/) - Common testing framework.
 * [Launch4j](http://launch4j.sourceforge.net/) - Wraps JARs in lightweight and native Windows executables.
+* [Quartz](https://github.com/quartz-scheduler/quartz) - a richly featured, open source job scheduling library that can be integrated within virtually any Java application - from the smallest stand-alone application to the largest e-commerce system.
```
Comment (akullpp) [inline README.md]: Begin the description with the first letter in uppercase. The description sounds more than an advertisement, please write something more informative.

File: README.md [hunk 2]
```diff
@@ -662,6 +663,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [GreenMail](http://www.icegreen.com/greenmail/) - In-memory email server for integration testing. Supports SMTP, POP3 and IMAP including SSL.
 * [Hamcrest](http://hamcrest.org/JavaHamcrest/) - Matchers that can be combined to create flexible expressions of intent.
 * [J8Spec](https://github.com/j8spec/j8spec/) - J8Spec is a library that allows tests written in Java to follow the BDD style introduced by RSpec and Jasmine.
+* [JBehave](http://jbehave.org/) - Framework for Behavioural Driven Development
```
Comment (akullpp) [inline README.md]: Missing the `.`

Comment (tracer021) [inline README.md]: Done with the change.

File: README.md (hunk 1 above)
Comment (tracer021) [inline README.md]: Done updating the description

**Pull request merged**